scripts/download_lyrics.py

```diff
@@ -26,24 +27,31 @@ def download_lyrics(api_key):
         "Dragonforce",
     ]
 
-    genius = lyricsgenius.Genius(api_key)
-
-    for band in bands:
+	genius = lyricsgenius.Genius(api_key)
+	
+	for band in bands:
         # We're using nested try/except blocks here for 2 reasons:
         # 1. the song downloading can fail from a connection error
         #    (could use a library like tenacity for retrying logic)
         # 2. song titles with forward slashes on them will not be saved
         #    (we can figure out how to handle this too)
-        try:
-            songs = genius.search_artist(band, sort="popularity", max_songs=50)
-            for song in songs:
-                try:
-                    song.save_lyrics(extension="txt")
-                except Exception:
-                    pass
-        except Exception:
-            pass
-
+	# 2.5 Since songs.save_lyrics tries to save all of them at once, if it fails due to issue 2, it won't try the ones after it. 
+		try:
+			songs = genius.search_artist(band, sort="popularity", max_songs=50)
+			try:
+				songs.save_lyrics(extension="txt")
+			except Exception:
+				print ("Couldn't save lyrics")    
+		except Exception:
+			print ("Couldn't find band")
+	#Move the lyrics file to lyrics folder
+	files = glob.glob('./lyrics_*')
+	try:
+		os.mkdir('lyrics')
+	except:
+		pass
+	for lyricsfile in files:
+		shutil.move('{0}'.format(lyricsfile), 'lyrics/{0}'.format(lyricsfile))
```

Comment:
A cool thing you can do in Python 3.6 and newer is f-strings. For example, this same line could be written as:

```python
shutil.move(f'{lyricsfile}', f'lyrics/{lyricsfile}')
```

You can check out more about it here: https://www.python.org/dev/peps/pep-0498/